setup.py

```diff
@@ -63,43 +66,92 @@
 
     'Sphinx',
 
-    # For the bodhi-client
-    'click',
-
     'WebOb>=1.4.1',
     ]
 
 if sys.version_info[:3] < (2,7,0):
-    requires.append('importlib')
+    server_requires.append('importlib')
 
 if sys.version_info[:3] < (2,5,0):
-    requires.append('pysqlite')
+    server_requires.append('pysqlite')
+
+
+setuptools.command.egg_info.manifest_maker.template = 'BODHI_MANIFEST.in'
+
 
 setup(name='bodhi',
-      version='2.1.9',
-      description='bodhi',
+      version=VERSION,
+      description='bodhi common package',
       long_description=README + '\n\n' +  CHANGES,
       classifiers=[
         "Programming Language :: Python",
-        "Framework :: Pyramid",
-        "Topic :: Internet :: WWW/HTTP",
-        "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
         ],
       author='',
       author_email='',
       url='',
-      keywords='web fedora pyramid',
-      packages=find_packages(),
+      keywords='fedora',
+      packages=['bodhi'],
       include_package_data=True,
       zip_safe=False,
-      install_requires = requires,
+      install_requires = [],
       tests_require = [
           'nose',
           'nose-cov',
           'webtest',
           'mock'
       ],
       test_suite="nose.collector",
+      )
+
+
+setuptools.command.egg_info.manifest_maker.template = 'CLIENT_MANIFEST.in'
+
+
+setup(name='bodhi-client',
+      version=VERSION,
+      description='bodhi client',
+      long_description=README + '\n\n' +  CHANGES,
+      classifiers=[
+        "Programming Language :: Python",
+        ],
+      author='',
+      author_email='',
+      url='',
+      keywords='fedora',
+      packages=['bodhi.client'],
+      include_package_data=False,
+      zip_safe=False,
+      install_requires = ['click'],
+      entry_points = """\
+      [console_scripts]
+      bodhi = bodhi.client:cli
+      """,
+      )
+
+
+setuptools.command.egg_info.manifest_maker.template = 'SERVER_MANIFEST.in'
+
+
+setup(name='bodhi-server',
+      version=VERSION,
+      description='bodhi server',
+      long_description=README + '\n\n' +  CHANGES,
+      classifiers=[
+        "Programming Language :: Python",
+        "Framework :: Pyramid",
+        "Topic :: Internet :: WWW/HTTP",
+        "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
```

Comment:
Note: there is a classifier to specify the license, I like to use it when I set the classifiers.

Entire list: https://pypi.python.org/pypi?%3Aaction=list_classifiers